src/test/java/hudson/plugins/audit_trail/ConfigurationAsCodeTest.java

```diff
@@ -0,0 +1,55 @@
+package hudson.plugins.audit_trail;
+
+import hudson.ExtensionList;
+import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
+import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Pierre Beitz
+ * on 2019-07-20.
+ */
+public class ConfigurationAsCodeTest {
+
+    @ClassRule
+    @ConfiguredWithCode("jcasc.yml")
+    public static JenkinsConfiguredWithCodeRule r = new JenkinsConfiguredWithCodeRule();
+
+    @Issue("JENKINS-57232")
+    @Test
+    public void should_support_configuration_as_code() {
```

Comment:
do you want to add an export test as well?  there's docs on that in the jcasc plugins guide

Reply:
good point, I'll add an export test!